src/prisma/schema.prisma

```diff
@@ -278,14 +278,8 @@ model ContractEventLogs {
   createdAt        DateTime @default(now())
   updatedAt        DateTime @updatedAt
 
-  @@id([transactionHash, logIndex])
-  @@index([timestamp])
-  @@index([blockNumber])
-  @@index([contractAddress])
-  @@index([topic0])
-  @@index([topic1])
-  @@index([topic2])
-  @@index([topic3])
+  @@id([chainId, blockNumber, transactionHash, logIndex])
+  @@index([timestamp, chainId, contractAddress])
```

Comment:
How are we querying the data? Did a quick scan, it seems like we don't always pass in a timestamp. In that case we'd probably want to start with `chainId`, `contractAddress` - then timestamp or blockNumber